src/views/ManageList.jsx

```diff
@@ -1,7 +1,80 @@
-export function ManageList() {
+import { addItem } from '../api/firebase';
+import { useRef } from 'react';
+
+export function ManageList({ listPath }) {
+	const formRef = useRef();
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		const formData = new FormData(e.target);
+		const itemName = formData.get('itemName');
+		const daysUntilNextPurchase = parseInt(
+			formData.get('daysUntilNextPurchase'),
+		);
+
+		try {
+			await addItem(listPath, {
+				itemName,
+				daysUntilNextPurchase,
+			});
+
+			alert('Item saved successfully');
+		} catch (error) {
+			alert('There was a problem');
```

Comment:
We don't necessarily have to fully display the `error` in this catch statement, but I do think we could make this error status more specific.

Reply:
I modified the catch block to display a more detailed error message.

```javascript
try {
...
} catch (error) {
   alert(`There was a problem: ${error.message}`);
};